dot/rpc/modules/dev_test.go

```diff
@@ -77,18 +79,28 @@ func TestDevModule_EpochLength(t *testing.T) {
 			},
 			args: args{
 				req: &EmptyRequest{},
-				res: &res,
 			},
+			exp: "0x1700000000000000",
 		},
 	}
 	for _, tt := range tests {
+		var res string
+		tt.args.res = &res
 		t.Run(tt.name, func(t *testing.T) {
 			m := &DevModule{
 				networkAPI:       tt.fields.networkAPI,
 				blockProducerAPI: tt.fields.blockProducerAPI,
 			}
-			if err := m.EpochLength(tt.args.r, tt.args.req, tt.args.res); (err != nil) != tt.wantErr {
+			var err error
+			if err = m.EpochLength(tt.args.r, tt.args.req, tt.args.res); (err != nil) != tt.wantErr {
 				t.Errorf("EpochLength() error = %v, wantErr %v", err, tt.wantErr)
+				return
+			}
+			if tt.wantErr {
+				assert.EqualError(t, err, tt.err.Error())
+			} else {
+				assert.NoError(t, err)
+				assert.Equal(t, tt.exp, *tt.args.res)
```

Comment:
You can probably move this after the `else` clause. If there is an error, it should not modify `t.args.res`, so the assertion should pass as well.

Also it's usually easier to have `tt.args.res` as a value, and pass it as pointer with `&tt.args.res` to the function `EpochLength`. I think you save a single *` character overall :exploding_head:

Reply:
Missed this one, but will make this change!